### PR TITLE
adding config for data store aat and perftest

### PR DESCRIPTION
**Pull request by mario-paniccia**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/aat/common/ccd/data-store-api.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ccd-data-store-api
+    version: 1.2.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ccd/data-store-api:latest
+      prometheus:
```
Comment (timja) [inline k8s/aat/common/ccd/data-store-api.yaml]: you don't need this prometheus config
```suggestion
```

File: k8s/aat/common/ccd/data-store-api.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        enabled: true
```
Comment (timja) [inline k8s/aat/common/ccd/data-store-api.yaml]:
```suggestion
```

File: k8s/perftest/common/ccd/data-store-api.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ccd-data-store-api
+    version: 1.2.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ccd/data-store-api:latest
+      prometheus:
```
Comment (timja) [inline k8s/perftest/common/ccd/data-store-api.yaml]:
```suggestion
```

File: k8s/perftest/common/ccd/data-store-api.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        enabled: true
```
Comment (timja) [inline k8s/perftest/common/ccd/data-store-api.yaml]:
```suggestion
```

File: k8s/perftest/common/ccd/data-store-api.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    global:
+      environment: aat
```
Comment (adusumillipraveen) [inline k8s/perftest/common/ccd/data-store-api.yaml]:
```suggestion
      environment: perftest
```

File: k8s/aat/common/ccd/data-store-api.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    global:
+      environment: perftest
```
Comment (adusumillipraveen) [inline k8s/aat/common/ccd/data-store-api.yaml]:
```suggestion
      environment: aat
```

**Pull request merged**